Source/Core/Core/Config/GraphicsSettings.h

```diff
@@ -126,6 +126,14 @@ extern const Info<bool> GFX_HACK_VERTEX_ROUDING;
 extern const Info<u32> GFX_HACK_MISSING_COLOR_VALUE;
 extern const Info<bool> GFX_HACK_FAST_TEXTURE_SAMPLING;
 
+// Graphics.Hacks EFB Scale exclusion settings
+static constexpr int GFX_EFB_SCALE_EXCLUDE_WIDTH_MAXIMUM = 640;
```

Comment:
Instead of hardcoding these, please use `EFB_WIDTH` and `EFB_HEIGHT`.

Reply:
Done, had to include VideoCommon/VideoCommon.h here for the variables to be accessible but seems safe (all unit tests pass, general testing OK, contents of the header seem to not conflict with what's here).

Reply:
Sorry, I should have been clearer.  I'd suggest getting rid of `GFX_EFB_SCALE_EXCLUDE_WIDTH_MAXIMUM` and the one for  height and instead using the `EFB_WIDTH` and `EFB_HEIGHT` where necessary.

Reply:
Cool, updated. I also enhanced the UI, more details in the most recent comment.

Reply:
Also, I've been force-pushing changes so as not to create a mess of superseding commits, please let me know if additional commits would be preferable. The latest changes affect many of the same classes when compared to the previous commit, should help with review.